### PR TITLE
Problem: rpc fails when basefee overflow int64

### PR DESCRIPTION
**Pull request by yihuang**
- add integration test to reproduce

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

## Summary by CodeRabbit

- **Bug Fixes**
	- Fixed an issue where base fees could exceed system limits, potentially causing errors.
- **New Features**
	- Introduced a new configuration setting for adjusting base fees and minimum gas prices in the `cronos_777-1` network.
- **Tests**
	- Added a test to verify JSON-RPC behavior when the base fee overflows int64.
- **Chores**
	- Modified the setup functionality in the test_rollback file to use a different initial block height parameter.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ### Bug Fixes
 
 - [#1336](https://github.com/crypto-org-chain/cronos/pull/1336) Update ethermint to develop to fix feeHistory rpc api.
+- [#]() Fix issues when basefee overflows int64
```
Comment (yihuang) [inline CHANGELOG.md:8]:
```suggestion
- [#1353](https://github.com/crypto-org-chain/cronos/pull/1353) Fix issues when basefee overflows int64
```

File: integration_tests/test_overflow.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+import pytest
+
+from .utils import ADDRS, KEYS, send_transaction, setup_custom_cronos, w3_wait_for_block
+
+
+@pytest.fixture(scope="module")
+def cronos(tmp_path_factory):
+    path = tmp_path_factory.mktemp("cronos-mempool")
```
Comment (yihuang) [inline integration_tests/test_overflow.py:10]:
```suggestion
    path = tmp_path_factory.mktemp("cronos-overflow")
```

**Review by mmsqe — COMMENTED** [review]
might update ethermint as well? and seems need set gas_prices when create validator

File: integration_tests/test_overflow.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        "from": ADDRS["validator"],
+        "to": ADDRS["community"],
+        "value": 1,
+        "gasPrice": w3.eth.gas_price,
+    }
+    tx["gas"] = w3.eth.estimate_gas(tx)
+    receipt = send_transaction(
+        w3,
+        tx,
+        KEYS["community"],
```
Comment (mmsqe) [inline integration_tests/test_overflow.py:34]: might align to use validator to sign and need more balance

**Review by mmsqe — COMMENTED** [review]
it seems a rare case to use [RPCMinGasPrice](https://github.com/crypto-org-chain/ethermint/blob/release/v0.21.x-cronos/rpc/backend/call_tx.go#L435) which is under `app-config.minimum-gas-prices`

**Pull request closed without merging**